include/loads_bcs/constraints.tcc

```diff
@@ -64,29 +55,21 @@ bool mpm::Constraints<Tdim>::assign_nodal_velocity_constraints(
 template <unsigned Tdim>
 bool mpm::Constraints<Tdim>::assign_nodal_frictional_constraint(
     int nset_id, const std::shared_ptr<mpm::FrictionConstraint>& fconstraint) {
-  bool status = false;
+  bool status = true;
```

Comment:
Do we need `bool` for this function, or `void` would suffice?
Here is where it is used: https://github.com/cb-geo/mpm/blob/8f19a4e1fae9a8f1afef0e0e5f723881008e3560/include/solvers/mpm_base.tcc#L863